[2023-06-04T14:00:20.838+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: open_brewery_pipeline.data_extraction manual__2023-06-04T14:00:17.783597+00:00 [queued]>
[2023-06-04T14:00:20.851+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: open_brewery_pipeline.data_extraction manual__2023-06-04T14:00:17.783597+00:00 [queued]>
[2023-06-04T14:00:20.853+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-06-04T14:00:20.873+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): data_extraction> on 2023-06-04 14:00:17.783597+00:00
[2023-06-04T14:00:20.880+0000] {standard_task_runner.py:57} INFO - Started process 202 to run task
[2023-06-04T14:00:20.890+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'open_brewery_pipeline', 'data_extraction', 'manual__2023-06-04T14:00:17.783597+00:00', '--job-id', '18', '--raw', '--subdir', 'DAGS_FOLDER/open_brewery_pipeline.py', '--cfg-path', '/tmp/tmpk8kcp4sl']
[2023-06-04T14:00:20.893+0000] {standard_task_runner.py:85} INFO - Job 18: Subtask data_extraction
[2023-06-04T14:00:20.962+0000] {task_command.py:410} INFO - Running <TaskInstance: open_brewery_pipeline.data_extraction manual__2023-06-04T14:00:17.783597+00:00 [running]> on host c59e3d6df99f
[2023-06-04T14:00:21.072+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='open_brewery_pipeline' AIRFLOW_CTX_TASK_ID='data_extraction' AIRFLOW_CTX_EXECUTION_DATE='2023-06-04T14:00:17.783597+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-06-04T14:00:17.783597+00:00'
[2023-06-04T14:00:21.076+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-06-04T14:00:21.077+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'source activate /opt/***/.venv/bin/activate python3 /opt/***/scripts/extraction.py']
[2023-06-04T14:00:21.094+0000] {subprocess.py:86} INFO - Output:
[2023-06-04T14:00:21.100+0000] {subprocess.py:93} INFO - /bin/bash: line 1: activate: No such file or directory
[2023-06-04T14:00:21.102+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2023-06-04T14:00:21.117+0000] {taskinstance.py:1824} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 211, in execute
    f"Bash command failed. The command returned a non-zero exit code {result.exit_code}."
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2023-06-04T14:00:21.124+0000] {taskinstance.py:1350} INFO - Marking task as FAILED. dag_id=open_brewery_pipeline, task_id=data_extraction, execution_date=20230604T140017, start_date=20230604T140020, end_date=20230604T140021
[2023-06-04T14:00:21.149+0000] {standard_task_runner.py:109} ERROR - Failed to execute job 18 for task data_extraction (Bash command failed. The command returned a non-zero exit code 1.; 202)
[2023-06-04T14:00:21.179+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 1
[2023-06-04T14:00:21.205+0000] {taskinstance.py:2651} INFO - 0 downstream tasks scheduled from follow-on schedule check
